src/Authenticator/ResultInterface.php

```diff
@@ -17,32 +17,32 @@
     /**
      * General Failure
      */
-    const FAILURE = 0;
+    const FAILURE = 'FAILURE';
 
     /**
      * Failure due to identity not being found.
      */
-    const FAILURE_IDENTITY_NOT_FOUND = -1;
+    const FAILURE_IDENTITY_NOT_FOUND = 'FAILURE_IDENTITY_NOT_FOUND';
 
     /**
-     * Failure due to invalid credential being supplied.
+     * Failure due to invalid credentials being supplied.
      */
-    const FAILURE_CREDENTIAL_INVALID = -2;
+    const FAILURE_CREDENTIALS_INVALID = 'FAILURE_CREDENTIALS_INVALID';
 
     /**
-     * Failure due to other circumstances.
+     * The authentication credentials were not found in the request.
      */
-    const FAILURE_OTHER = -3;
+    const FAILURE_CREDENTIALS_NOT_IN_REQUEST = 'FAILURE_CREDENTIALS_NOT_IN_REQUEST';
```

Comment:
What’s the reason behind that change?

Reply:
I did move around the lines, I did not replace those. Take a look at the full diff.

Reply:
@inoas I'm talking about `FAILURE_CREDENTIALS_NOT_FOUND` => `FAILURE_CREDENTIALS_NOT_IN_REQUEST`